src/applications/find-forms/tests/e2e/00-required.cypress.spec.js

```diff
@@ -45,10 +53,11 @@ describe('functionality of Find Forms', () => {
     cy.get(`${SELECTORS.SEARCH_RESULT_TITLE}`);
 
     // iterate through all pages and ensure each form download link is present on each form result.
-    const validForms = stub.data.filter(form => form.attributes.validPdf);
-    const sortedForms = sortBy(validForms, 'id');
+    const validForms = stub.data
+      .filter(form => form.attributes.validPdf)
+      .sort((a, b) => sortTheResults(SORT_OPTIONS[INITIAL_SORT_STATE], a, b));
     const pageLength = 10;
-    const pages = chunk(sortedForms, pageLength);
+    const pages = chunk(validForms, pageLength);
```

Comment:
Maybe we could add onto this test by changing the sort option in the select menu and then performing some assertions?

Reply:
Good call!